cmd/operator/main.go

```diff
@@ -14,5 +14,45 @@
 
 package main
 
+import (
+	"context"
+	"time"
+
+	"github.com/GoogleCloudPlatform/cloud-run-release-operator/internal/run"
+	"github.com/GoogleCloudPlatform/cloud-run-release-operator/pkg/config"
+	"github.com/GoogleCloudPlatform/cloud-run-release-operator/pkg/rollout"
+	format "github.com/logrusorgru/aurora"
+	log "github.com/sirupsen/logrus"
+)
+
 func main() {
+	config := &config.Config{
+		Metadata: &config.Metadata{
+			Project: "gtvo-test",
+			Service: "hello",
+			Region:  "us-east1",
+		},
+		Rollout: &config.Rollout{
+			Steps: []int64{5, 30, 80},
+		},
+	}
+	log := log.New()
+
+	client, err := run.NewAPIClient(context.TODO(), config.Metadata.Region)
+	if err != nil {
+		log.Fatal("could not initilize Cloud Run client", err)
+	}
+	roll := rollout.New(client, config, log)
+
+	svc, err := roll.Manage()
```

Comment:
don't return svc.

Reply:
I agree that it might be better not to return a full service to the user. I have two concerns about this, though

1) We still need to return something since `nil` svc means that nothing changed but that there was no errors (e.g. no candidate exists)
2) The returned service is useful for testing that the changes happened as expected. We could solve this by writing a a function that calls `Manage`. Something like:
```
func (r *Rollout) Run(...) (someStruct, error) {
   svc, err := r.Manage(...)
}
```
The Rollout instance would then just call `Run` instead of manage and we can still test that the service was changed as expected. However, I don't know if you think that this is the best approach.

Any thoughts?

Reply:
Most idiomatic way to do this:

`changed, err := roll.RolloutService(svc)`